docs/tools/terraform/howto/vnet-peering-azure.rst

```diff
@@ -0,0 +1,213 @@
+Azure virtual network peering
+=============================
+
+This how-to is based on original :doc:`article </docs/platform/howto/vnet-peering-azure>`
+made for Aiven and Azure cli.
+It follows original chapter-by-chapter,
+I will find every step signed with the very same title below in the example.
+
+And while most of terraform manifestos can be applied in one go,
+we'll have to break it up into two:
+
+1. First, we will create most of the necessary resources
+2. Then we will configure Azure provider using data from step one
+   to create the last resource and connect networks together
+
+Before you start
+~~~~~~~~~~~~~~~
+
+Create an  `Aiven authentication token </docs/platform/howto/create_authentication_token>`.
+Then set up `authentication for Azure <https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#authenticating-to-azure>`_
+and `Azure Active Directory <https://registry.terraform.io/providers/hashicorp/azuread/latest/docs#authenticating-to-azure-active-directory>`_.
+For example:
+
+.. code-block::
+
+    terraform {
+      required_providers {
+        aiven = {
+          source = "aiven/aiven"
+          verstion = ">= 3.8.0, < 4.0.0"
+        }
+        azuread = {
+          source  = "hashicorp/azuread"
+          version = "=2.30.0"
+        }
+        azurerm = {
+          source  = "hashicorp/azurerm"
+          version = "=3.30.0"
+        }
+      }
+    }
+
+    provider "aiven" {
+      api_token = var.aiven_api_token
+    }
+
+    provider "azuread" {
+      client_id     = "00000000-0000-0000-0000-000000000000"
+      client_secret = var.azure_client_secret
+      tenant_id     = "00000000-0000-0000-0000-000000000000"
+    }
+
+    provider "azurerm" {
+      features {}
+      subscription_id = "00000000-0000-0000-0000-000000000000"
+      client_id       = "00000000-0000-0000-0000-000000000000"
+      client_secret   = var.azure_client_secret
+      tenant_id       = "00000000-0000-0000-0000-000000000000"
+    }
+
+
+1. Create or bind the resources
```

Comment:
```suggestion
Step 1: Create or bind the resources
```

Reply:
I think this format will be accepted by the linter (I used it in another article).